foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
```

Comment:
This is standard fedora packaging versioning scheme for "from SCM" aka "snapshot" which is I believe better than just number:

https://fedoraproject.org/wiki/Packaging:Versioning